week4/main.cpp

```diff
@@ -0,0 +1,73 @@
+//CodeU:
+//
+//  Created by Julia Semavina on 15.06.2017
+//
+//  problem specification:
+//      You are given a 2-dimensional map of tiles. Each tile is either land or water.
+//      You have to write a function that counts the number of islands.
+//
+//  comments:
+//      DFS
+//
+
+
+#include <iostream>
+#include <vector>
+
+
+void DFS(int x, int y, int IslandSize, std::vector<std::vector<bool>> &Island,
```

Comment:
Here I think we could use Island.size() instead of trusting the user giving us the correct size.